docs/how-to-guides/eagleye-integration-guide.md

```diff
@@ -0,0 +1,115 @@
+# Using Eagleye in Autoware
+
+This page will show you how to set up [Eagleye](https://github.com/MapIV/eagleye) in order to use it with Autoware.
+For the details of the integration proposal, please refer to [this](https://github.com/orgs/autowarefoundation/discussions/3257) Discussion.
+
+## What is Eagleye?
+
+Eagleye is an open-source GNSS/IMU-based localizer initially developed by [MAP IV. Inc](https://map4.jp/).
+Eagleye provides a cost-effective alternative to LiDAR and point cloud-based localization by using low-cost GNSS and IMU sensors to provide vehicle position, orientation, and altitude information.  
+Using GNSS Doppler velocity, Eagleye can also compensate for wheel speed and IMU angular velocity bias to achieve more accurate speed and heading estimation.
+
+With the addition of Eagleye integration into Autoware, users will be able to choose between their existing LiDAR and point cloud-based localization stacks or GNSS/IMU-based Eagleye localizer, depending on their specific needs and operating environment.
+
+There are two ways to utilize Eagleye results in the Autoware localization stack:
+
+1. Feed only twist into the EKF localizer.
+
+![Eagleye twist integration](images/eagleye-integration-guide/eagleye_twist.drawio.svg)
```

Comment:
This is probably my mistake in my previous suggestion. Currently, the numbered list is still incorrect. It's still 1 and then 1.
![Screenshot from 2023-04-12 15-44-52](https://user-images.githubusercontent.com/23651560/231374282-21e62f58-1eee-434a-9353-67a349711652.png)
With the following changes (please note the added spaces),
```suggestion
   ![Eagleye twist integration](images/eagleye-integration-guide/eagleye_twist.drawio.svg)
```
the results should be as follows.
![Screenshot from 2023-04-12 15-50-39](https://user-images.githubusercontent.com/23651560/231374896-241eda39-0b34-45cb-a047-87a8deda9909.png)

Reply:
Fixed

Reply:
![image](https://user-images.githubusercontent.com/18298605/231896788-163e5771-09f8-489d-912e-f24c3daa539e.png)

Reply:
Thank you! There’s one small thing. I think you misspelled Autoware as Autowares in the index.md. The CI is failing.